lpeg/io_modules/decoders/logfmt.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Logfmt Message Decoder Module
+
+## Decoder Configuration Table for input cfg file
+decoder_module = "decoders.logfmt"
+
+-- Allows you to remap parsed logfmt tags into another field
+logfmt_conf = {
+    -- override root fields using Fields fields
+    remap_hash_root = {
```

Comment:
Even if i clearly am agreeing with you here, real life shows that you sometimes can't get people/code to conform to these ideals.

To me it has proved handy and even mandatory in sadly too many setups. I'm even adding this feature in a few lua plugins.

But if you prefer not to encourage these practices by dropping this feature it's fine by me.

wdyt ?